app/src/main/java/org/oppia/android/app/devoptions/DeveloperOptionsActivity.kt

```diff
@@ -0,0 +1,27 @@
+package org.oppia.android.app.devoptions
+
+import android.content.Context
+import android.content.Intent
+import android.os.Bundle
+import org.oppia.android.app.activity.InjectableAppCompatActivity
+
+class DeveloperOptionsActivity : InjectableAppCompatActivity() {
```

Comment:
KDoc missing. Every public facing functions and classes should have relevant KDoc explanations. Ditto elsewhere.

Reply:
Got it.